src/app/pages/AddAppPage/components/add-app/add-app.component.ts

```diff
@@ -306,4 +322,20 @@ export class AddAppComponent implements OnInit, OnChanges {
         this.needsToUpdateNgoLogo = false;
 
     }
+
+    private setFormDefaults() {
+        this.setDefaultsForLogoRelated();
+        this.selectedAppTags = [];
+        this.selectedTechnologies = [];
+    }
+
+    private static scrollToTop() {
+        (function scrollSmoothly() {
+            const currentScroll = document.documentElement.scrollTop || document.body.scrollTop;
+            if (currentScroll > 0) {
+                window.requestAnimationFrame(scrollSmoothly);
+                window.scrollTo(0, currentScroll - (currentScroll / 8));
+            }
+        })();
```

Comment:
a little bit hacky. can you use scrollIntoView instead? there's neat example in approve-app.component (line 85) that method could be extracted overall and used anywhere we need it

Reply:
@y0rg1 can you please check? thanks :smile:

Reply:
@lrdTnc I'm not entirely sure how to implement this cleanly using scrollIntoView because the navbar component (or any other top page component) is outside add-app's hierarchy.
I went with this approach in order to enable smooth scrolling on Edge and IE browsers as well.